go.mod

```diff
@@ -7,7 +7,7 @@ toolchain go1.21.0
 require (
 	github.com/aws/aws-sdk-go v1.51.31
 	github.com/google/go-cmp v0.6.0
-	github.com/grafana/grafana-aws-sdk v0.27.0
+	github.com/grafana/grafana-aws-sdk v0.28.1-0.20240716142732-adf2cef95fa0
```

Comment:
should we publish a new version or something? Sorry if this me being not knowledgeable about go lol.

Reply:
For sure! This is just so CI could run while the release is pending.